app/bundles/CampaignBundle/Views/Campaign/list.html.twig

```diff
@@ -0,0 +1,150 @@
+{% set isIndex = tmpl == 'index' ? true : false %}
+
+{# @todo #}
+{#% extends isIndex ? 'MauticCoreBundle:Standard:index.html.twig' : 'MauticCoreBundle:Default:content.html.twig' %#}
```

Comment:
`MauticCoreBundle:Standard:index.html.twig` Not Found

Reply:
For these templates that are outside of the bundle, do you want to have these included in this PR or would you like to have a separate PR for those?

Reply:
Include them in so we don't have to go back and do another round of refactoring. It will also help with other bundles.